src/d3a/setup/two_sided_market/one_cep_one_load_immediate_match.py

```diff
@@ -0,0 +1,58 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from pendulum import duration
+from d3a.models.appliance.switchable import SwitchableAppliance
+from d3a.models.area import Area
+from d3a.models.strategy.load_hours import LoadHoursStrategy
+from d3a.models.appliance.simple import SimpleAppliance
+from d3a.models.strategy.commercial_producer import CommercialStrategy
+from d3a_interface.constants_limits import ConstSettings
+
+
+def get_setup(config):
+    ConstSettings.IAASettings.MARKET_TYPE = 2
+
+    area = Area(
+        'Grid',
+        [
+            Area(
+                'House 1',
+                [
+                    Area('H1 General Load', strategy=LoadHoursStrategy(
+                        avg_power_W=200,
+                        hrs_per_day=24,
+                        hrs_of_day=list(range(24)),
+                        initial_buying_rate=35,
+                        final_buying_rate=35,
+                        update_interval=duration(minutes=14)
```

Comment:
Very minor but it could be squashed into less number of lines.

Reply:
Let me try to change it, this suggestion is contrary to the rest of the setup files, but I'll do it.

Reply:
IMO: Please leave it as is. I personally prefer to scroll through a list where each line has one setting than multiple.

Reply:
Since I also tend to share @hannesdiedrich 's opinion, I left it as is.